server/tracker.go

```diff
@@ -27,6 +27,14 @@ const (
 
 	// firstPoolNum is the starting number for pools
 	firstPoolNum = 1
+
+	// log buckets
+	logBan           = "[Ban] "
+	logBlame         = "[Blame] "
+	logBroadcast     = "[Broadcast] "
+	logCommunication = "[Communication] "
+	logDirectMessage = "[DirectMessage] "
+	logListener      = "[Listener] "
```

Comment:
These are pretty helpful for picking out specific things in the log. Not sure if I chose the right buckets though.

Reply:
Should add [Announce] prefix from comment above.

Reply:
Wasn't sure if making a one-time bare string a constant would be the right way. Thanks for the guidance.  f8a49a2

Reply:
I like consistency. Now all these are in the same place. =)